src/main.ts

```diff
@@ -4,15 +4,35 @@ import OpenAI from "openai";
 import { Octokit } from "@octokit/rest";
 import parseDiff, { Chunk, File } from "parse-diff";
 import minimatch from "minimatch";
+import { Certificate } from "crypto";
 
 const GITHUB_TOKEN: string = core.getInput("GITHUB_TOKEN");
 const OPENAI_API_KEY: string = core.getInput("OPENAI_API_KEY");
 const OPENAI_API_MODEL: string = core.getInput("OPENAI_API_MODEL");
+const OPENAI_API_VERSION: string = core.getInput("OPENAI_API_VERSION");
+const OPENAI_BASE_URL: string = core.getInput("OPENAI_BASE_URL"); // Keep the default value as undefined instead of empty strings.
+
+// Supports HTTP requests debugging via an environment variable.
+const debugHttp: string | undefined = process.env.DEBUG_HTTP;
```

Comment:
Very handy to debug the openai library calls and see what was wrong while calling Azure instead of OpenAI.